requirements/edx/base.in

```diff
@@ -93,7 +93,6 @@ futures ; python_version == "2.7"   # via django-pipeline, python-swift-client,
 glob2==0.3                          # Enhanced glob module, used in openedx.core.lib.rooted_paths
 gunicorn==17.5
 help-tokens
-html5lib==0.999                     # HTML parser, used for capa problems
```

Comment:
I think we want to keep a line for html5lib, because it's a direct dependency of the capa code in this repository.  I was just suggesting that it shouldn't be pinned to an exact version here.  As we gradually get individual dependencies up to date, this file will become mostly a list of package names with no version constraints.

Reply:
Oh I see! Sorry about that.

Reply:
This is now fixed :D